src/routes/(app)/+layout.svelte

```diff
@@ -55,6 +55,103 @@
       height: 100%;
       min-height: 100%;
       background-color: #f6f8fa;
+
+    }
+
+    :root {
```

Comment:
I know this is probably just for testing purposes, but we won't actually commit all these properties to the project. By default, we won't use the custom properties at all. We will just list them in docs or maybe have an example file of their use.

Plus, I think it would be best if you test this with Storybook, rather than the test app here. So, you could copy and paste these into this storybook file: https://github.com/ForgeRock/forgerock-web-login-framework/blob/beta/.storybook/preview-head.html#L11

Reply:
I do want to capture all of these converted color values, though. That will save some time for us and our customers if they want to just tweak the existing colors.

Reply:
yeah awesome, i didn't know i could use storybooks for it.